core/src/main/java/io/aiven/klaw/controller/OperationalRequestsController.java

```diff
@@ -42,6 +44,36 @@ public ResponseEntity<ApiResponse> createConsumerOffsetsResetRequest(
         HttpStatus.OK);
   }
 
+  @PostMapping(
+      value = "/operationalRequest/approve/reqId/{reqId}",
+      produces = {MediaType.APPLICATION_JSON_VALUE})
```

Comment:
This could go either way but what do you think of

 "/operationalRequest/reqId/{reqId}/approve",
 "/operationalRequest/reqId/{reqId}/decline",
 "/operationalRequest/reqId/{reqId}/delete",
 
 So it is {requestType}/{requestId}/{action on that request}

Reply:
Yea sometimes i see 'action' before or after.  But it looks good. changed.